lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -50,7 +52,13 @@ Creates object instance of the class
 
 =item * C<redis> - Redis client object (must be compatible with L<Mojo::Redis2>). This argument will override the C<redis_uri> argument.
 
-=item * C<timeout> - Request timeout, in seconds. If not set, uses the environment variable C<RPC_QUEUE_RESPONSE_TIMEOUT>, or defaults to 300
+=item * C<timeout> - Request timeout, in seconds. If not set, uses the environment variable C<RPC_QUEUE_RESPONSE_TIMEOUT>, or defaults to 30
+
+=item * C<queue_separation_enabled> - Boolean to specify if queue separation should be enabled or not.
```

Comment:
Better to explain what "queue separation" is, rather than just repeating the name.